psql-users.sh

```diff
@@ -0,0 +1,2 @@
+psql -U postgres -c "create user test with superuser password 'test';"
```

Comment:
Is this script being run? I don't see anything invoking it... It also looks like we're configuring Postgres not to verify passwords during testing... So maybe we don't need this.

The Postgres TCP port isn't being exposed outside the host, right? Pretty sure it isn't, just want to confirm.

Reply:
Since this script is being mounted on the psql container under "/docker-entrypoint-initdb.d/psql-users.sh" it is ran as an init script. Looking at the .travis.yml again it seems we only need the db and not the user for this particular repository. 

TCP port is not being exposed but since the containers are sharing network namespace it is available over localhost. For docker-compose it is done by setting  "network_mode: "service:postgres"

Reply:
Okay, I see. Cool.